cognite_toolkit/_cdf_tk/utils/interactive_select.py

```diff
@@ -135,8 +131,16 @@ def select_hierarchies_and_data_sets(self) -> tuple[list[str], list[str]]:
                 return [], [data_set]
             selected_hierarchies = self._select(f"hierarchies in data set {data_set!r} ", hierarchies, True, "multiple")
             return selected_hierarchies or [], [data_set]
-        else:
+
+    def select_hierarchies_or_data_sets(self) -> Literal["Hierarchy", "Data Set"]:
```

Comment:
Moved this out in a separate method for easy reuse - this is a small refactoring.